sites/fast-website/src/app/components/side-navigation/side-navigation.ts

```diff
@@ -20,21 +20,50 @@ export class SideNavigation extends FASTElement {
         x => x.header !== "Github"
     );
 
+    public scrollFunc = (num1: number, num2: number): void => window.scrollTo(num1, num2);
+
     public clickHandler = (e: Event, parent: boolean): void => {
         const target: HTMLElement = e.target as HTMLElement;
```

Comment:
You can use `e.currentTarget` and not have to bother with the `parent` targeting below.

Reply:
Thank you very much! I have updated this accordingly.